src/frontend-admin/src/services/AxiosService/AxiosService.ts

```diff
@@ -0,0 +1,135 @@
+import axios, {AxiosInstance} from 'axios'
+//import qs from 'qs' ??
+//import buildQuery from 'odata-query' ??
+import decode from 'jwt-decode'
+
+export interface IUserInfo {
+    name: string
+    token: string
```

Comment:
Though it's not so concise I'd recommend calling this thing `accessToken` just to avoid any and all confusion.